addons/html_builder/static/tests/helpers.js

```diff
@@ -174,6 +174,13 @@ export async function openBuilderSidebar() {
     await animationFrame();
 }
 
+export function addPlugin(Plugin) {
+    registry.category("website-plugins").add(Plugin.id, Plugin);
+    after(() => {
```

Comment:
Question: what does `after` do exactly ?

Reply:
The function passed to after will be executed at the end of the test. In this case, it cleans up the registry.